openlabcmd/openlabcmd/zk.py

```diff
@@ -297,24 +271,23 @@ def update_service(self, service_name, role, n_type, alarmed=None,
             if not isinstance(alarmed, bool):
                 raise exceptions.ValidationError('alarmed should be boolean '
                                                  'value.')
-            old_service['alarmed'] = alarmed
-            old_service['alarmed_at'] = current_time
+            old_service.alarmed = alarmed
+            old_service.alarmed_at = current_time
```

Comment:
I saw the current_time  using the local time zone. Right now the running host is UTC, so we just in the same time zone. How about use the same UTC time zone with specified UTC?

Reply:
`updated_at` and `created_at` are generated by zookeeper internally, it uses local time. So that to keep the same with zookeeper, we should use local time as well.

Reply:
emm, imaging that Cloud A using time zone A locally, Cloud B using time zone B locally, and Cloud C using time zone C. If we deploy zk cluster on those, which time zone it will use?  I;m afraid we may maintain multiple time zone format. Allright, let's forget this now, as it's not a real problem. We can leave it on our notes.

Reply:
This is a good point. ++ for UTC, since different cloud may live in different time zone. I'll fix it in the next pr. Thanks!